### DISCUSSION TITLE
Add --start and --end to elastalert-test-rule

### DISCUSSION THREAD

**Discussion by pboushy** [jertel/elastalert2#1198] [Ideas]
The main `elastalert` command has a `--start` and `--end` that are really helpful for testing frequency rules.

It would be great if `elastalert-test-rule` had the same capability.

**Comment by jertel** [comment]
Per the usage it does have the same capability. Are you seeing otherwise?

```
$ elastalert-test-rule 
usage: elastalert-test-rule [-h] [--schema-only] [--days DAYS] [--start START] [--end END] [--stop-error] [--formatted-output] [--data FILENAME] [--alert] [--save-json FILENAME] [--use-downloaded] [--max-query-size MAX_QUERY_SIZE]
                            [--count-only] [--config CONFIG]
                            rule
elastalert-test-rule: error: the following arguments are required: rule
```

**Reply by pboushy** [reply]
You're right, it does exist...

I didn't check the command help because I read this in [the documentation] (https://elastalert2.readthedocs.io/en/latest/ruletypes.html#testing) (emphasis mine):
> --days N: Instead of the default 1 day, query N days. **For selecting more specific time ranges, you must run ElastAlert 2 itself and use --start and --end.**

Maybe I should submit feature requests after getting some sleep.

I'll submit a PR to modify that to say "For selecting more specific time ranges, use --start and --end."
